app/assets/stylesheets/views/_worldwide-organisation.scss

```diff
@@ -6,9 +6,13 @@
 .js-enabled {
   .worldwide-organisation-header {
     .show-other-content {
-      @include govuk-font(14);
+      all: inherit;
+      display: inline;
       margin-left: .3em;
```

Comment:
Suggestion: adding `cursor: pointer` would make the mouse hover behaviour consistent with the other links here.

Reply:
Good spot, thanks, I've now added that.